pkgs/tools/misc/svu/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, buildGoModule, fetchFromGitHub, testers, svu }:
+
+buildGoModule rec {
+  pname = "svu";
+  version = "1.10.2";
+
+  src = fetchFromGitHub {
+    owner = "caarlos0";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "37AT+ygN7u3KfFqr26M9c7aTt15z8m4PBrSd+G5mJcE=";
+  };
+
+  vendorHash = "sha256-+e1oL08KvBSNaRepGR2SBBrEDJaGxl5V9rOBysGEfQs=";
+
+  ldflags =
+    [ "-s" "-w" "-X main.version=${version}" "-X main.builtBy=nixpkgs" ];
+
```

`fetchFromGitHub` uses the archive tarball by default, which doesn't have any git information, alternatively you can set `leaveDotGit = true;` in `fetchFromGitHub` and add git to `nativeCheckInputs` if you think these tests are important

```suggestion

  # test assumes source directory to be a git repository
  postPatch = ''
    rm internal/git/git_test.go
  '';

```

---

does it have `git` though? If so, I can (in a future version) change the tests so they create a test repo (which is probably better anyway)

---

> does it have git though?

It does only if you add it